src/error.rs

```diff
@@ -111,6 +111,9 @@ pub enum Error {
         container_name: String,
         violation: String,
     },
+
```

Comment:
And another one *(insert DJ Khaled meme here)*. For now this is necessary, but I started working on refactoring the `operator-rs` error situation. So that we finally have proper module level errors.

Reply:
Yeah, absolutely agreed. Was thinking the same while adding this error